src/Features/CSharp/Portable/InlineParameterNameHints/InlineParameterNameHintsService.cs

```diff
@@ -23,26 +23,17 @@ namespace Microsoft.CodeAnalysis.CSharp.InlineParameterNameHints
     /// as well as associate the adornments back to the parameter name
     /// </summary>
     [ExportLanguageService(typeof(IInlineParameterNameHintsService), LanguageNames.CSharp), Shared]
-    internal class InlineParameterNameHintsService : IInlineParameterNameHintsService
+    internal class InlineParameterNameHintsService : AbstractInlineParameterNameHintsService
```

Comment:
once we have a VB and C# version we rename thsee.  so this should be CSharpInlineParameterNameHintsService.  That way it's easy to find/nav-to them since we have distinct names, not three types with the same name.